examples/starter/Readme.md

```diff
@@ -0,0 +1,94 @@
+# Example - A basic example to start with the MongoDB Atlas and Terraform
+
+This project aims to provide a very straight-forward example of setting up Terraform with MongoDB Atlas.
+
+
+## Dependencies
+
+* Terraform v0.13
+* A MongoDB Atlas account - provider.mongodbatlas: version = "~> 0.6.5"
+
+## Usage
+
+**1\. Ensure your MongoDB Atlas credentials are set up.**
+
+This can be done using environment variables:
+
+```bash
+export MONGODB_ATLAS_PUBLIC_KEY="xxxx"
+export MONGODB_ATLAS_PRIVATE_KEY="xxxx"
+```
+
+... or follow as in the `variables.tf` file and create **terraform.tfvars** file with all the variable values and make sure **not to commit it**.
+
+**2\. Review the Terraform plan.**
+
+Execute the below command and ensure you are happy with the plan.
+
+``` bash
+$ terraform plan
+```
+This project currently does the below deployments:
+
+- Atlas Project
+- MongoDB cluster - M10
+- Database User
+- IP Whitelist
+
+**3\. Configure the security group as required.**
+
+The security group in this configuration allows All Traffic access in Inbound and Outbound Rules.
```

Comment:
Perhaps from another one?  Probably want to change this to have them set an IP they want to access from w/ the new access list resources.

Reply:
removed this.